aria/modeling/models.py

```diff
@@ -0,0 +1,283 @@
+
+# endregion
+
+
+# region service instance models
+
+class Service(aria_declarative_base, service_instance.ServiceBase):
+    pass
+
+
+class Node(aria_declarative_base, service_instance.NodeBase):
+    pass
+
+
+class Group(aria_declarative_base, service_instance.GroupBase):
+    pass
+
+
+class Policy(aria_declarative_base, service_instance.PolicyBase):
+    pass
+
+
+class Substitution(aria_declarative_base, service_instance.SubstitutionBase):
+    pass
+
+
+class SubstitutionMapping(aria_declarative_base, service_instance.SubstitutionMappingBase):
+    pass
+
+
+class Relationship(aria_declarative_base, service_instance.RelationshipBase):
+    pass
+
+
+class Capability(aria_declarative_base, service_instance.CapabilityBase):
+    pass
+
+
+class Interface(aria_declarative_base, service_instance.InterfaceBase):
+    pass
+
+
+class Operation(aria_declarative_base, service_instance.OperationBase):
+    pass
+
+
+class Artifact(aria_declarative_base, service_instance.ArtifactBase):
+    pass
+
+# endregion
+
+
+# region service changes models
+
+class ServiceUpdate(aria_declarative_base, service_changes.ServiceUpdateBase):
+    pass
+
+
+class ServiceUpdateStep(aria_declarative_base, service_changes.ServiceUpdateStepBase):
+    pass
+
+
+class ServiceModification(aria_declarative_base, service_changes.ServiceModificationBase):
+    pass
+
+# endregion
+
+
+# region common service models
+
+class Parameter(aria_declarative_base, service_common.ParameterBase):
+    pass
+
+
+class Type(aria_declarative_base, service_common.TypeBase):
+    pass
+
+
+class Metadata(aria_declarative_base, service_common.MetadataBase):
+    pass
+
+
+class PluginSpecification(aria_declarative_base, service_common.PluginSpecificationBase):
+    pass
+
+# endregion
+
+
+# region orchestration models
+
+class Execution(aria_declarative_base, orchestration.ExecutionBase):
+    pass
+
+
+class Plugin(aria_declarative_base, orchestration.PluginBase):
+    pass
+
+
+class Task(aria_declarative_base, orchestration.TaskBase):
+    pass
+
+
+class Log(aria_declarative_base, orchestration.LogBase):
+    pass
+
+# endregion
+
+
+models_to_register = [
+    # Service template models
+    ServiceTemplate,
+    NodeTemplate,
+    GroupTemplate,
+    PolicyTemplate,
+    SubstitutionTemplate,
+    SubstitutionTemplateMapping,
+    RequirementTemplate,
+    RelationshipTemplate,
+    CapabilityTemplate,
+    InterfaceTemplate,
+    OperationTemplate,
+    ArtifactTemplate,
+
+    # Service instance models
+    Service,
+    Node,
+    Group,
+    Policy,
+    SubstitutionMapping,
+    Substitution,
+    Relationship,
+    Capability,
+    Interface,
+    Operation,
+    Artifact,
+
+    # Service changes models
+    ServiceUpdate,
+    ServiceUpdateStep,
+    ServiceModification,
+
+    # Common service models
+    Parameter,
+    Type,
+    Metadata,
+    PluginSpecification,
+
+    # Orchestration models
+    Execution,
+    Plugin,
+    Task,
+    Log
+]
+
+__all__ = (
```

Comment:
I wanted this to be close to `models_to_register` because they are almost the same, but I will do it your way. I am adding comments though so people will know to update the other list.